multiproject/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = "0.1.0rc"
```

Comment:
We can also go straight to 1.0rc1 here. We usually start semver wrong, but 0.x is for initial development. We are arguable skipping initial development, and going straight to a stable public API. This would indicate a 1.0 release.

Anyways, just making sure we don't release a 0.1.0 as the first release. `0.1.0rc` sounds like our first release is 0.1.0

```suggestion
__version__ = "1.0.0rc1"
```